vm/systemSmartContracts/delegation.go

```diff
@@ -377,6 +393,40 @@ func (d *delegation) delegateUser(
 	return vmcommon.Ok
 }
 
+func (d *delegation) makeStakeArgsIfAutomaticActivation(
+	config *DelegationConfig,
+	status *DelegationContractStatus,
+	globalFund *GlobalFundData,
+) [][]byte {
+	if !config.AutomaticActivation || len(status.NotStakedKeys) == 0 {
```

Comment:
shouldn't we add some extra cost for this function call?

Reply:
no - we have gasCost usage in auction per staked key.

Reply:
ok